cvcreator/txt2yaml.py

```diff
@@ -152,6 +153,39 @@ def main():
                     a["Tools"] = line[6:]
                 else:
                     a[cur] += " " + line
+    
+        elif header[0] == "B":
+            
+            header = "B%d" % (int(header[1:-1])+1)
+            data["Publications"][header] = b = {}
+
+            if line[:8] == "Journal":
```

Comment:
More idiomatic:
```suggestion
            entries = ["Journal", "Date", "Authors", "Title", "Summary", "DOI"]
            for entry in entries:
                if line.startswith(entry):
                    cur = entry
                    b[entry] = line[len(entry):]
                    break
            else:
                a[cur] += " " + line
```
Are you sure it should be `a` in the last line there?